<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QWidget" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="35" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="okBtn">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>
OK
</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="2">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>InkBox settings</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="nextBtn">
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="previousBtn">
         <property name="text">
          <string>Previous</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="Line" name="line_4">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="31" column="0">
      <widget class="Line" name="line_5">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="32" column="0">
      <widget class="QPushButton" name="aboutBtn">
       <property name="text">
        <string>About InkBox</string>
       </property>
      </widget>
     </item>
     <item row="33" column="0">
      <widget class="Line" name="line">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="30" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="26" column="0">
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout_11">
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="7" column="0">
            <widget class="QCheckBox" name="clockCheckBox">
             <property name="text">
              <string>Clock: Show seconds</string>
             </property>
            </widget>
           </item>
           <item row="15" column="0">
            <widget class="Line" name="line_7">
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="16" column="0">
            <layout class="QGridLayout" name="gridLayout_4">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="2">
              <widget class="QPushButton" name="usbmsBtn">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Go</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Enter USB Mass Storage session</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="6" column="0">
            <widget class="Line" name="line_6">
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="Line" name="line_8">
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <layout class="QGridLayout" name="gridLayout_9">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="screenRefreshLabel">
               <property name="text">
                <string>Screen refresh</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="2">
              <widget class="QComboBox" name="comboBox">
               <item>
                <property name="text">
                 <string>Every page</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1 page</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2 pages</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3 pages</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4 pages</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5 pages</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>6 pages</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Never refresh</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item row="18" column="0">
            <widget class="Line" name="line_3">
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QCheckBox" name="quoteCheckBox">
             <property name="text">
              <string>Disable authors quotes</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QCheckBox" name="menuBarCheckBox">
             <property name="text">
              <string>Always show status bar</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <family>Chivo</family>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Reading</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <layout class="QGridLayout" name="gridLayout_6">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="1" column="1">
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="wordsNumberLabel">
               <property name="text">
                <string>Words number per page</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QSpinBox" name="wordsNumber">
               <property name="minimum">
                <number>20</number>
               </property>
               <property name="maximum">
                <number>300</number>
               </property>
               <property name="singleStep">
                <number>20</number>
               </property>
               <property name="value">
                <number>20</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>Chivo</family>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Home</string>
             </property>
            </widget>
           </item>
           <item row="14" column="0">
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>Chivo</family>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Storage</string>
             </property>
            </widget>
           </item>
           <item row="17" column="0">
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>Chivo</family>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Networking</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QCheckBox" name="demoCheckBox">
             <property name="text">
              <string>Disable &quot;Welcome to InkBox&quot; message</string>
             </property>
            </widget>
           </item>
           <item row="19" column="0">
            <layout class="QGridLayout" name="gridLayout_3">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Request DHCP lease</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="requestLeaseBtn">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Request</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="8" column="0">
            <layout class="QGridLayout" name="gridLayout_8">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="uiScalingLabel">
               <property name="text">
                <string>UI scaling factor</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSlider" name="uiScalingSlider">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>2</number>
               </property>
               <property name="singleStep">
                <number>1</number>
               </property>
               <property name="pageStep">
                <number>1</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="uiScaleNumberLabel">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>1</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="5" column="0">
            <layout class="QGridLayout" name="gridLayout_13">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="1">
              <spacer name="horizontalSpacer_9">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="setPasscodeBtn">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Set</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="setPasscodeLabel">
               <property name="text">
                <string>Set a passcode</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="softwareLabel">
             <property name="font">
              <font>
               <family>Chivo</family>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>System</string>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="9" column="0">
            <widget class="QCheckBox" name="darkModeCheckBox">
             <property name="text">
              <string>Enable night mode</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="securityLabel">
             <property name="font">
              <font>
               <family>Chivo</family>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Security</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="Line" name="line_9">
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <layout class="QGridLayout" name="gridLayout_7">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="verticalSpacing">
              <number>0</number>
             </property>
             <item row="0" column="2">
              <widget class="QPushButton" name="updateBtn">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Update</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="updateLabel">
               <property name="text">
                <string>Update InkBox</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="10" column="0">
            <layout class="QGridLayout" name="gridLayout_12">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Reset this device</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <spacer name="horizontalSpacer_8">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="resetBtn">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Reset</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <widget class="Line" name="line_2">
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QCheckBox" name="enableLockscreenCheckBox">
             <property name="text">
              <string>Enable lock screen and passcode</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
